clang/lib/Parse/ParseStmt.cpp

```diff
@@ -571,11 +571,8 @@ StmtResult Parser::ParseExprStatement(ParsedStmtContext StmtCtx) {
   }
 
   Token *CurTok = nullptr;
-  // If the semicolon is missing at the end of REPL input, consider if
-  // we want to do value printing. Note this is only enabled in C++ mode
-  // since part of the implementation requires C++ language features.
   // Note we shouldn't eat the token since the callback needs it.
-  if (Tok.is(tok::annot_repl_input_end) && Actions.getLangOpts().CPlusPlus)
+  if (Tok.is(tok::annot_repl_input_end))
```

Comment:
Hi,

Landed here as was studying a bit on `annot_repl_input_end`

Curious if this change also calls for a change in the docs ? @vgvassilev 

I see the following 

![image](https://github.com/user-attachments/assets/d6e22a0a-9c17-4859-a1cf-ea09124b6226)

Reply:
Probably.

Reply:
Cool going ahead and raising a PR for the same !